src/core/utils/urlutils.js

```diff
@@ -55,3 +55,24 @@ export function addParamsToUrl (url, params = {}) {
   }
   return url.split('?')[0] + '?' + urlParams;
 }
+
+/**
+ * returns if two SearchParams objects have the same key,value entries
+ * @param {SearchParams} params1
+ * @param {SearchParams} params2
+ * @return {boolean} true if params1 and params2 have the same key,value entries, false otherwise
+ */
+export function equivalentParams (params1, params2) {
```

Comment:
Would it make sense for there to be an "isEqual" function in the "SearchParams" class?

Reply:
sadly SearchParams returns either a native implementation or our own implementation. Thus, we would only have this `isEqual` function if we were on an unsupported browser (e.g. IE11).

Reply:
Oh I didn't know that 😮  in that case, this works!